### DISCUSSION TITLE
Is the storage temp folder automatically emptied?

### DISCUSSION THREAD

**Discussion by carolineboeykens** [craftcms/cms#13007] [Q&A] [answered]
We have issues with projects where the storage > runtime > temp folder size is getting large.
We thought this folder was getting cleared automatically but it does not seem to happen?

![Screenshot 2023-03-29 at 12 10 01](https://user-images.githubusercontent.com/42932673/228501829-ffcd7b6e-5ca2-4d20-8f1d-ab9285f06fbc.png)

Does anyone have an idea what the logic is here?
We have now a temporarily solution with a cron job that is clearing this folder every week.

**Comment by brandonkelly** [comment, marked as answer]
The directory is cleared by the Caches utility (“Temp files” option), or via the following commands;

```
> php craft clear-caches/all
> php craft clear-caches/temp-files
```

Craft doesn’t have the means to run commands on a schedule (yet), but you can set up your Cron job to run that `clear-caches/temp-files` command.
